From 61c9bce74a4a58998b54fee9e28094daae209d0d Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Fri, 11 Jan 2013 19:38:39 +0100
Subject: [PATCH][BUILD] Build API support.


Signed-off-by: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
---
 bootstrap |   13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/bootstrap b/bootstrap
index 9a4f796..2aa3630 100755
--- a/bootstrap
+++ b/bootstrap
@@ -584,12 +584,25 @@ while test $# != 0; do
   --version) cmake_version_display ; exit 2 ;;
   --verbose) cmake_verbose=TRUE ;;
   --enable-ccache) cmake_ccache_enabled=TRUE ;;
+  --build=*) ;;
+  --libdir=*) ;;
+  --bindir=*) ;;
+  --sbindir=*) ;;
+  --sysconfdir=*) ;;
+  --localstatedir=*) ;;
+  --includedir=*) ;;
+  --libexecdir=*) ;;
+  --infodir=*) ;;
+  --disable-static|--disable-silent-rules) ;;
   --) shift; break ;;
   *) die "Unknown option: $1" ;;
   esac
   shift
 done
 
+cmake_data_dir=$(echo $cmake_data_dir | sed -e "s,$cmake_prefix_dir,,g")
+cmake_man_dir=$(echo $cmake_man_dir | sed -e "s,$cmake_prefix_dir,,g")
+
 # If verbose, display some information about bootstrap
 if [ -n "${cmake_verbose}" ]; then
   echo "---------------------------------------------"
-- 
1.7.10.4

