From e811d9fa7982546fa8dc38f9069774daca2e16c4 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Sat, 3 Aug 2013 10:38:02 +0200
Subject: [PATCH] configure: Turn -nomake and -make into autoconf style
 arguments

So we can easily build libcommuni with mauibuild.

diff --git a/configure b/configure
index 66b2e5a..64fab1d 100755
--- a/configure
+++ b/configure
@@ -110,7 +110,7 @@ while [ "$#" -gt 0 ]; do
         INSTALL_FEATURES="$VAL"
         ;;
     nomake)
-        case "$1" in
+        case "$VAL" in
         examples)
             BUILD_EXAMPLES=no
             shift
@@ -126,7 +126,7 @@ while [ "$#" -gt 0 ]; do
         esac
         ;;
     make)
-        case "$1" in
+        case "$VAL" in
         examples)
             BUILD_EXAMPLES=yes
             ;;
-- 
1.8.1.4

