From 2d59c3f0cef7891d80c48a2404bfb35c0a61fe39 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sun, 16 Sep 2012 12:24:06 -0400
Subject: [PATCH] supplicant: Install systemd units if configured to do so

---
 wpa_supplicant/.config  |  1 +
 wpa_supplicant/Makefile | 14 ++++++++++++++
 2 files changed, 15 insertions(+)

diff --git a/wpa_supplicant/.config b/wpa_supplicant/.config
index 1c2f527..3d7f538 100644
--- a/wpa_supplicant/.config
+++ b/wpa_supplicant/.config
@@ -2,6 +2,7 @@ CONFIG_CTRL_IFACE=y
 CONFIG_CTRL_IFACE_DBUS=y
 CONFIG_CTRL_IFACE_DBUS_NEW=y
 CONFIG_CTRL_IFACE_DBUS_INTRO=y
+CONFIG_SYSTEMD=y
 CONFIG_DRIVER_WEXT=y
 CONFIG_LIBNL32=y
 CONFIG_DRIVER_NL80211=y
diff --git a/wpa_supplicant/Makefile b/wpa_supplicant/Makefile
index 540909e..da826b4 100644
--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -1236,6 +1236,20 @@ DBUS_CFLAGS += -DCONFIG_DBUS
 DBUS_OBJS += dbus/dbus_common.o
 endif
 
+ifdef CONFIG_SYSTEMD
+SYSTEMD_SYSTEM_UNITDIR = $(shell $(PKG_CONFIG) --variable=systemdsystemunitdir systemd)
+SYSTEMD_SYSTEM_UNITS = \
+	systemd/wpa_supplicant.service \
+	systemd/wpa_supplicant@.service \
+	systemd/wpa_supplicant-nl80211@.service \
+	systemd/wpa_supplicant-wired@.service
+
+install-data::
+	for sv in $(SYSTEMD_SYSTEM_UNITS); do \
+	  install -D $${sv} $(DESTDIR)$(SYSTEMD_SYSTEM_UNITDIR)/$$(basename $${sv}); \
+	done
+endif
+
 OBJS += $(DBUS_OBJS)
 CFLAGS += $(DBUS_CFLAGS)
 LIBS += $(DBUS_LIBS)
-- 
1.7.11.4

