From 1d267d39c8b93ee59e9d26ee5148d6e92a79814a Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 21 May 2013 23:13:56 +0200
Subject: [PATCH 4/4] supplicant: Install systemd units if configured to do so

---
 wpa_supplicant/.config  |    1 +
 wpa_supplicant/Makefile |   14 ++++++++++++++
 2 files changed, 15 insertions(+)

diff --git a/wpa_supplicant/.config b/wpa_supplicant/.config
index ae76ce4..441eef3 100644
--- a/wpa_supplicant/.config
+++ b/wpa_supplicant/.config
@@ -2,6 +2,7 @@ CONFIG_CTRL_IFACE=y
 CONFIG_CTRL_IFACE_DBUS=y
 CONFIG_CTRL_IFACE_DBUS_NEW=y
 CONFIG_CTRL_IFACE_DBUS_INTRO=y
+CONFIG_SYSTEMD=y
 CONFIG_DRIVER_WEXT=y
 CONFIG_LIBNL32=y
 CONFIG_DRIVER_NL80211=y
diff --git a/wpa_supplicant/Makefile b/wpa_supplicant/Makefile
index 505aa1f..f05780d 100644
--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -1300,6 +1300,20 @@ DBUS_CFLAGS += -DCONFIG_DBUS
 DBUS_OBJS += dbus/dbus_common.o
 endif
 
+ifdef CONFIG_SYSTEMD
+SYSTEMD_SYSTEM_UNITDIR = $(shell $(PKG_CONFIG) --variable=systemdsystemunitdir systemd)
+SYSTEMD_SYSTEM_UNITS = \
+	systemd/wpa_supplicant.service \
+	systemd/wpa_supplicant@.service \
+	systemd/wpa_supplicant-nl80211@.service \
+	systemd/wpa_supplicant-wired@.service
+
+install-data::
+	for sv in $(SYSTEMD_SYSTEM_UNITS); do \
+		install -D $${sv} $(DESTDIR)$(SYSTEMD_SYSTEM_UNITDIR)/$$(basename $${sv}); \
+	done
+endif
+
 OBJS += $(DBUS_OBJS)
 CFLAGS += $(DBUS_CFLAGS)
 LIBS += $(DBUS_LIBS)
-- 
1.7.10.4

